cachito/workers/pkg_managers/gomod.py

```diff
@@ -187,7 +189,54 @@ def resolve_gomod(app_source_path, request, dep_replacements=None):
             )
             shutil.copytree(tmp_download_cache_dir, str(bundle_dir.gomod_download_dir))
 
-        return module, deps
+        # package level dependencies
```

Comment:
Could add a log statement here stating that you're getting the package dependencies?